practice/earthquake/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <link rel="stylesheet"
+    href="http://openlayers.org/en/v3.17.1/css/ol.css" type="text/css">
+    <script src="http://openlayers.org/en/v3.17.1/build/ol.js"
+     type="text/javascript"></script>
+    <title>Earthquake</title>
+    <link rel="stylesheet" href="site.css">
+  </head>
+  <body>
+    <h2>Earthquake!</h2>
+    <div id="map" class="map">
+      Loading...
+    </div>
+    <script src="https://code.jquery.com/jquery-2.2.3.js"></script>
+    <script src="site.js"></script>
+    <script src="http://cdn.jsdelivr.net/lodash/4.11.2/lodash.js"></script>
```

Comment:
You'll have to manually resolve dependencies. Right now if `site.js` used lodash `_`, there would be an error because lodash hasn't been parsed yet.